iam-token-manager/v1.ts

```diff
@@ -92,6 +93,8 @@ export class IamTokenManagerV1 {
       // tslint:disable-next-line
       console.log(CLIENT_ID_SECRET_WARNING);
     }
+
+    this.requestWrapperInstance = new RequestWrapper({});
```

Comment:
What are your thoughts on altering the constructor so that if no params are passed in, it defaults to `{}`? I hate to make the user pass in an empty object if they aren't providing any parameters